planting/planting_api_v1.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env python
-# encoding: utf-8
+# -*- coding: utf -*-
```

Comment:
it's a mistake

Reply:
its edit by mark, who said its a better style of coding

Reply:
ok